tests/MongoDB.EntityFrameworkCore.FunctionalTests/Storage/MongoDatabaseCreatorTests.cs

```diff
@@ -76,7 +76,7 @@ public void EnsureDeleted_returns_true_and_deletes_database_when_it_exists()
     [Fact]
     public void EnsureDeleted_returns_false_when_it_does_not_exist()
     {
-        var database = new TemporaryDatabaseFixture();
+        using var database = new TemporaryDatabaseFixture();
```

Comment:
Should we inject the fixture instead?

Reply:
No, this test deletes the database. That would break the other tests.

Reply:
Just keeping the database creation/deletion tests isolated to avoid concurrency issues.